playground.angular.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "command": ["npm", "run", "dev.gsc"],
+    "title": "Stencil Components",
+    "defaultStatus": ["🧩", "🧩"],
+    "status": {
+      "rebuild failed": ["🚨", "🚨"],
+      "rebuild finished": ["🧩", "🧩"]
+    }
+  },
+  {
+    "command": ["npm", "run", "dev.gscc"],
+    "title": "Stencil Chart Components",
+    "defaultStatus": ["🧩", "🧩"],
+    "status": {
+      "rebuild failed": ["🚨", "🚨"],
+      "rebuild finished": ["🧩", "🧩"]
+    }
+  },
+  {
+    "command": ["npm", "run", "dev.spark"],
+    "title": "Spark Package",
+    "defaultStatus": ["✨", "✨"]
+  },
+  {
+    "command": ["npm", "run", "dev.examples"],
+    "title": "Docs - http://localhost:8080",
+    "defaultStatus": ["🗃️", "🗃️"],
+    "status": {
+      "ERROR": ["🚨", "🚨"],
+      "compiled successfully": ["🗃️", "🗃️"]
+    }
+  },
+  {
+    "command": ["npm", "run", "playground.angular.start"],
```

Comment:
We could just run all playgrounds instead of each individual one in a separate script like I do here. Thoughts?

Reply:
I think running one playground at a time would probably work best.